### ISSUE TITLE
Stick to the bottom of the view port?

### ISSUE THREAD

**Issue by pentateu** [yahoo/react-stickynode#47]
Hi,

I'm currently trying to figure it out how to stick to the bottom of the page/viewport. 

Is that possible?

Cheers

**Comment by hankhsiao** [comment]
That is possible, but why not you can just make it fixed? Can you talk
about your scenario?

**Comment by gavinhekogan** [comment]
@pentateu Assign a class name to activeClass props
e.g.
```
activeClass={stick-to-bottom}
.stick-to-bottom {
  top: initial !important;
  bottom: 0;
}
```
Ideally shouldn't have to use important on css, or you can modify lib to accept props like stickToBottom={true}, then remove `top`, and add `bottom: 0;`

**Comment by infacq** [comment]
@hankhsiao do you mind to show example how to make it stick to the bottom and release when approach the specified `boundaryBottom`

**Comment by NicoleRauch** [comment]
I'm also interested in such a behaviour. Here is my use case:

I want to build a long order summary page which should have a button "Buy this!" at the bottom of the page that should be always visible (aka. "sticky"). Below the button there is a general footer that should become visible if the user scrolls all the way down (then the button area should lose its stickiness and move upwards with the rest of the page).

Can you please show me how to achieve this with react-stickynode?

**Comment by hankhsiao** [comment]
I don't think a way to achieve your need without modifying code. Maybe you
can set topOffset close to bottom?

**Comment by NicoleRauch** [comment]
@hankhsiao Thanks for the pointer! I've experimented with this and this is what I've achieved by now:

- I've set the top offset to the window bottom, and it sticks to the bottom of the page. (Good!)
- I've set the bottomBoundary to where the general footer starts, and it scrolls up when the general footer comes into view. (Good!)
- But when I scroll down again, the element does not become sticky again, it just scrolls down with the rest of the page. But what I would like to achieve is this: When the element touches the bottom of the window again, it should become sticky again and not scroll down and out of sight.

Any idea on how I can make it sticky again?

**Comment by NicoleRauch** [comment]
Oh, and one more question, if I may: I've moved the React element for the "Buy this!" button to the top of the page (because otherwise I do not see it when I am at the top of the page because it only seems to be moved down by react-stickynode, not up).
But when I open the page, the button is right where I inserted it into the page, i.e. somewhere near the top. It is only when I start to scroll that the button moves to the bottom of the page. How can I tell it to be at the bottom right from the beginning? I think the "enabled" property is true by default, so there is no need to set this myself, I take it? But how can I tell it to move to its position initially already?
